TLM/TLM/TrafficManagerMod.cs

```diff
@@ -14,6 +14,12 @@ namespace TrafficManager {
     using UnityEngine.SceneManagement;
 
     public class TrafficManagerMod : IUserMod {
+#if BENCHMARK
+        public TrafficManagerMod() {
+            Benchmark.BenchmarkManager.Setup();
```

Comment:
Will this impair hot reloads? // cc: @kianzarrin

Reply:
I'm not sure but BENCHMARK is separate build config so that code will be removed when you use any other config

Reply:
it did not consider hot reloads TBH, but I don't think that it makes sense to hot reload during a benchmark, or risk the benchmark numbers due to hotreload left overs etc.
my idea is that you develop with the debug / release solution config, no benchmark will run in that case so you can hot reload as much as you like.
once you are done with your code changes you close the game, switch to benchmark, rebuild, then load you benchmark savegame with whatever setup to get the numbers.
just an idea though, should be discussed how you plan to do it.

Reply:
> I'm not sure but BENCHMARK is separate build config so that code will be removed when you use any other config

yes, that was the plan.

Reply:
Ah, good point!